File: 2240-number-of-ways-to-buy-pens-and-pencils/2240-number-of-ways-to-buy-pens-and-pencils.java
class Solution {
    public long waysToBuyPensPencils(int total, int cost1, int cost2) {
        long res=0;
        while(total >= 0){
            res += ((total/cost2) + 1);
            total -= cost1;
        }
        return res;
    }
}

File: 2240-number-of-ways-to-buy-pens-and-pencils/2240-number-of-ways-to-buy-pens-and-pencils.py
from math import floor
class Solution:
    def waysToBuyPensPencils(self, total: int, cost1: int, cost2: int) -> int:
        ans = 0
        y = lambda x: (total-x*cost1)/cost2
        for i in range(total+1):
            c = floor(y(i))
            if c>=0:
                ans += c+1
        return ans